scripts/commands/checkPagesRender.ts

```diff
@@ -0,0 +1,149 @@
+      default: false,
+      description: "Check the pages in the `api/qiskit/release-notes` folder.",
+    })
+    .option("translations", {
+      type: "boolean",
+      default: false,
+      description: "Check the pages in the `translations/` subfolders.",
+    })
+    .parseSync();
+};
+
+zxMain(async () => {
+  const args = readArgs();
+  await validateDockerRunning();
+  const files = await determineFilePaths(args);
+
+  let allGood = true;
+  let numFilesChecked = 1;
+  for (const fp of files) {
+    const rendered = await canRender(fp);
+    if (!rendered) {
+      console.error(`❌ Failed to render: ${fp}`);
+      allGood = false;
+    }
+
+    // This script can be slow, so log progress every 10 files.
+    if (numFilesChecked % 10 == 0) {
+      console.log(`Checked ${numFilesChecked} / ${files.length} pages`);
+    }
+    numFilesChecked++;
+  }
+
+  if (allGood) {
+    console.info("✅ All pages render without crashing");
+  } else {
+    console.error(
+      "💔 Some pages crash when rendering. This is usually due to invalid syntax, such as forgetting " +
+        "the closing component tag, like `</Admonition>`. You can sometimes get a helpful error message " +
+        "by previewing the docs locally or in CI. See the README for instructions.",
+    );
+    process.exit(1);
+  }
+});
+
+async function canRender(fp: string): Promise<boolean> {
+  const url = pathToUrl(fp);
+  try {
+    const response = await fetch(url);
+    if (response.status >= 300) {
+      return false;
+    }
+  } catch (error) {
+    return false;
+  }
+
+  return true;
+}
+
+function pathToUrl(path: string): string {
+  const strippedPath = path
+    .replace("docs/", "")
+    .replace("translations/", "")
+    .replace(/\.(?:md|mdx|ipynb)$/g, "");
+  return `http://localhost:${PORT}/${strippedPath}`;
+}
+
+async function validateDockerRunning(): Promise<void> {
+  try {
+    const response = await fetch(`http://localhost:${PORT}`);
+    if (response.status !== 404) {
+      console.error(
+        "Failed to access http://localhost:3000. Have you started the Docker server with `./start`? " +
+          "Refer to the README for instructions.",
+      );
+      process.exit(1);
+    }
+  } catch (error) {
+    console.error(
+      "Error when accessing http://localhost:3000. Make sure that you've started the Docker server " +
+        "with `./start`. Refer to the README for instructions.\n\n" +
+        `${error}`,
+    );
+    process.exit(1);
+  }
+}
+
+async function determineFilePaths(args: Arguments): Promise<string[]> {
+  const globs = ["docs/**/*.{ipynb,md,mdx}"];
+  if (!args.currentApis) {
+    globs.push("!docs/api/{qiskit,qiskit-ibm-provider,qiskit-ibm-runtime}/*");
+  }
+  if (!args.historicalApis) {
+    globs.push(
+      "!docs/api/{qiskit,qiskit-ibm-provider,qiskit-ibm-runtime}/[0-9]*/*",
+    );
+  }
+  if (!args.qiskitReleaseNotes) {
+    globs.push("!docs/api/qiskit/release-notes/*");
+  }
+  if (!args.translations) {
```

Comment:
Wait no, this was correct.

```suggestion
  if (!args.translations) {
```

Reply:
🤦 sorry